webbpsf/optics.py

```diff
@@ -709,26 +711,27 @@ class NIRCam_BandLimitedCoron(poppy.BandLimitedCoron):
     # Offsets along the bar occulters are based on
     # outputs from John Stansberry's filt_baroffset.pro
     # See https://github.com/mperrin/webbpsf/issues/63
-    offset_swb = {"F182M": -1.856,
-                  "F187N": -1.571,
-                  "F210M": -0.071,
-                  "F212N": 0.143,
-                  "F200W": 0.232,
-                  'narrow': -8.00}
+    # EDIT: updated on 8 Dec 2021 to grab offsets directly from pySIAF
+    # offset_swb = {"F182M": -1.856,
+    #               "F187N": -1.571,
+    #               "F210M": -0.071,
+    #               "F212N": 0.143,
+    #               "F200W": 0.232,
+    #               'narrow': -8.00}
```

Comment:
Add a comment of new values from current pysiaf version. For instance, v0.13.0 gives:
```python
{'F182M': -1.742781239401549,
 'F187N': -1.5437104345990476,
 'F210M': -0.03369518613985725,
 'F212N': 0.14397080350195612,
 'F200W': 0.19604610088955918,
 'narrow': -8.05266403068413}
```

Reply:
Obviously, @mperrin's suggestions override mine, here.

Reply:
I guess I'm curious what value you think it would have to include the numerical values as a comment text? Just for reference later?  If anything, I'd say it's worth putting a table of these into JDox, but that's out of scope for this PR :-)

Reply:
My thinking was that if the old ones were going to be commented out, we should add updated ones (with a version attachment) for reference as a comparison for clarity on how things changed.

Reply:
But removing all reference to the numbers is fine with me. A JDox table is the appropriate place to track this.